gallery/visualtests/nxBreadcrumb.js

```diff
@@ -0,0 +1,95 @@
+      manyLongSegmentsExample = '#nx-breadcrumb-many-long-segments-example .gallery-example-live',
+      oneSegmentExample = '#nx-breadcrumb-one-segment-example .gallery-example-live',
+      fourSegmentsExample = '#nx-breadcrumb-four-segments-example .gallery-example-live',
+      fiveSegmentsExample = '#nx-breadcrumb-five-segments-example .gallery-example-live';
+
+  it('looks right in the simple case', simpleTest(simpleExampleSelector));
+  it('looks right with long overflowing segments but no dropdown', simpleTest(longSegmentsExample));
+  it('looks right with long overflowing segments but no dropdown in a smaller container',
+      simpleTest(constrainedWidthExample));
+  it('looks right with a dropdown', simpleTest(manySegmentsExample));
+  it('looks right with long segments and a dropdown', simpleTest(manyLongSegmentsExample));
+  it('renders nothing with one segment', simpleTest(oneSegmentExample));
+  it('renders no dropdown with four segments', simpleTest(fourSegmentsExample));
+  it('renders a dropdown with five segments', simpleTest(fiveSegmentsExample));
+
+  it('looks right with a narrow dropdown', async function() {
+    const [example, dropdownBtn] = await waitAndGetElements(
+        manySegmentsExample,
+        `${manySegmentsExample} .nx-icon-dropdown__toggle`
+    );
+
+    await dropdownBtn.click();
+
+    await checkScreenshot(example, 848, 364);
+  });
+
+  it('looks right with a wide dropdown', async function() {
+    const [example, dropdownBtn] = await waitAndGetElements(
+        manyLongSegmentsExample,
+        `${manyLongSegmentsExample} .nx-icon-dropdown__toggle`
+    );
+
+    await dropdownBtn.click();
+
+    await checkScreenshot(example, 848, 396);
+  });
+
+  it('shows overflow tooltips on inline links', async function() {
+    const [example, link] = await waitAndGetElements(
+        longSegmentsExample,
+        `${longSegmentsExample} a`
+    );
+
+    await link.hover();
+    await wait(500);
+
+    const { x, y, width, height } = await example.boundingBox();
+
+    await checkScreenshotCoordinates(x, y - 21, width, height + 21);
+  });
+
+  it('shows overflow tooltips on dropdown links', async function() {
+    const [example, dropdownBtn] = await waitAndGetElements(
+        manyLongSegmentsExample,
+        `${manyLongSegmentsExample} .nx-icon-dropdown__toggle`
+    );
+
+    await dropdownBtn.click();
+
+    const [link] = await waitAndGetElements(`${manyLongSegmentsExample} .nx-dropdown-link`);
+    await link.hover();
+    await wait(500);
+
+    await checkScreenshot(example, 848, 396);
+  });
+
+  describe('in NxGlobalHeader', function() {
+    const { checkFullPageScreenshot, a11yTest, getPage } = setupBrowser('#/NxBreadcrumbGlobalHeaderExample', false);
+
+    it('looks right', async function() {
+      await getPage().setViewport({ width: 1366, height: 800 });
+      await checkFullPageScreenshot();
+    });
+
+    it('passes a11y checks', a11yTest(null, true));
```

Comment:
Out of curiosity, is the `a11Test` testing the `NxBreadcrumbGlobalHeaderExample` page without calling `getPage().setViewport({ width: 1366, height: 800 });` here?

Reply:
> Out of curiosity, is the a11Test testing the NxBreadcrumbGlobalHeaderExample page without calling getPage().setViewport({ width: 1366, height: 800 }); here?

Yes, the custom viewport size on the visual test was just to avoid being wasteful with a needlessly humongous image.  Our full-page screenshots are quite tall by default now

Reply:
Got it, yes the default viewport hight is 3000 by default.